Runtime/AvatarCreator/Scripts/WebRequests/AuthAPIRequests.cs

```diff
@@ -60,23 +68,23 @@ public async Task SendCodeToEmail(string email, string userId = "",CancellationT
 
             var payload = AuthDataConverter.CreatePayload(data);
 
-            var response = await webRequestDispatcher.SendRequest<ResponseText>($"{rpmAuthBaseUrl}/auth/start", HttpMethod.POST, headers, payload, ctx:cancellationToken);
+            var response = await webRequestDispatcher.SendRequest<ResponseText>($"{rpmAuthBaseUrl}auth/request-login-code", HttpMethod.POST, headers, payload, ctx:cancellationToken);
             response.ThrowIfError();
         }
 
         public async Task<UserSession> LoginWithCode(string code, string userIdToMerge = null, CancellationToken cancellationToken = default)
         {
             var body = new Dictionary<string, string>
             {
-                { AuthConstants.AUTH_TYPE_CODE, code }
+                { AuthConstants.CODE, code }
             };
             if (userIdToMerge != null)
             {
                 body.Add(AuthConstants.USER_ID, userIdToMerge);
             }
             var payload = AuthDataConverter.CreatePayload(body);
 
-            var response = await webRequestDispatcher.SendRequest<ResponseText>($"{rpmAuthBaseUrl}/auth/login", HttpMethod.POST, headers, payload, ctx:cancellationToken);
+            var response = await webRequestDispatcher.SendRequest<ResponseText>($"{rpmAuthBaseUrl}auth/login", HttpMethod.POST, headers, payload, ctx:cancellationToken);
```

Comment:
does this login payload now have the appName sent as well?

Reply:
🤔It does not. I wonder how the  editor worked. Maybe it had a user to merge and that one had the app authorized. Have to check it again.

Reply:
here it came from subdomain before probably
`string.Format(Env.RPM_SUBDOMAIN_BASE_URL, domain);`

Reply:
I think it worked when I tested with my changes as well but will go over it again anyway